yaml-tests/src/test/resources/encrypted.yamsql

```diff
@@ -0,0 +1,67 @@
+#
+# encrypted.yamsql
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2025 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+options:
+    supported_version: 4.5.6.0
+    connection_options:
+      ENCRYPT_WHEN_SERIALIZING: true
+      ENCRYPTION_PASSWORD: SQL
+---
+schema_template:
+    create table t(id bigint, s string, primary key(id))
+---
+setup:
+  steps:
+    - query: INSERT INTO t
+        VALUES (1, 'abc'),
+               (2, 'def')
+---
+test_block:
+  name: read-encrypted
+  options:
+    connection_options:
+      ENCRYPT_WHEN_SERIALIZING: true
+      ENCRYPTION_PASSWORD: SQL
+  tests:
+    -
+      - query: SELECT * FROM t
```

Comment:
Because of the way that requests are load-balanced across the versions, a better chance of hitting all servers will be achieved by adding more queries to the test (more than one, that is).
So for example, one more write and then read, or just more reads.